.circleci/config.yml

```diff
@@ -125,26 +113,59 @@ commands:
                 kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/liveapi-binary-com liveapi-binary-com=${DOCKHUB_ORGANISATION}/binary-static-liveapi:${CIRCLE_SHA1}
               fi
             done
-
+  notify_slack:
+    description: "Notify slack"
+    steps:
+      - slack/status:
+          include_project_field: false
+          failure_message: "Release failed for liveapi.binary.com with version *$(cat lib/version)*"
```

Comment:
This might be a bug if the build fails before `deploy` step

Reply:
@gokula-krishna-dev we only send notification for releases. and we link the CI job, so we can check why it fails.what is the issue?